cohere/client.py

```diff
@@ -287,17 +287,24 @@ def chat(
                 >>> print(res.text)
                 >>> print(res.citations)
         """
-        if chatlog_override is not None:
+        if chat_history is not None:
+            for entry in chat_history:
+                if "text" in entry:
```

Comment:
do you want to do `entry["message"] = entry["text"]` here?

Reply:
The endpoint still supports `text`, so we can send the entry with `text` instead of `message`. However, I agree that this attribution might make the transition easier later on.